frontend/src/components/Sidebar.js

```diff
@@ -0,0 +1,97 @@
+import { NavLink } from "react-router-dom";
+import Wrapper from "../styles/Sidebar";
+import {
+  HistoryIcon,
+  HomeIcon,
+  LibIcon,
+  LikeIcon,
+  SubIcon,
+  TrendingIcon,
+  VidIcon,
+} from "./Icons";
+import SidebarAuth from "./SidebarAuth";
+
+function Sidebar({ isSidebarOpen }) {
+  return (
+    <Wrapper open={isSidebarOpen}>
+      <NavLink
+        end
+        to="/"
+        className={({ isActive }) => (isActive ? "active" : "")}
```

Comment:
Appreciation:- Good that you are using `Navlink` and `isActive` prop in the sidebar.